### PR TITLE
Add more details to the Transforms section #1061

### PR DESCRIPTION
**Pull request by skaterdav85**
See issue #1061

### CODE REVIEW

File: source/models/defining-models.md [hunk 1]
```diff
@@ -78,9 +78,27 @@ export default DS.Model.extend({
...
+```app/models/person.js
+export default DS.Model.extend({
+  name: DS.attr('string'),
+  age: DS.attr('number'),
+  admin: DS.attr('boolean'),
+  birthday: DS.attr('date')
+});
+```
+
+The `string` and `number` transforms simply use the `String` and `Number`
```
Comment (locks) [inline source/models/defining-models.md]: Not sure about this sentence.

File: source/models/defining-models.md [hunk 2]
```diff
@@ -78,9 +78,27 @@ export default DS.Model.extend({
...
+functions respectively to coerce the attributes.
+
+The `boolean` transform can handle values other than `true` or
```
Comment (locks) [inline source/models/defining-models.md]: The boolean conditions being split in two sentences is confusing. First all the truthy values, then all the falsey values.

**Comment by skaterdav85** [comment]
Thanks for the feedback @locks. I made the changes.

**Comment by michaelrkn** [comment]
Thanks @skaterdav85! For the sake of brevity, what do you think about taking out lines 69-79? They feel a bit redundant with the more fleshed-out example you've provided.

**Comment by skaterdav85** [comment]
@michaelrkn I agree. I modified it a bit.

**Comment by michaelrkn** [comment]
Awesome! Can you just squash the commits down now?

**Comment by skaterdav85** [comment]
This is the first time I've rebased so let me know if I did it incorrectly =)

**Comment by michaelrkn** [comment]
Welcome to the wild world of the rebase! What you're looking for is under the [Commits tab](https://github.com/emberjs/guides/pull/1067/commits) for there to be only one commit. So, you're not quite there yet :) If you get stuck, feel free to jump on the documentation slack channel and see if somebody might be lurking who can help you out (possible even me!).

**Comment by acorncom** [comment]
@skaterdav85 Thanks for the work on this.  And agreed, one of us would be happy to help out on the rebasing stuff if you're around at some point on Slack ...

**Comment by skaterdav85** [comment]
I made a new PR: https://github.com/emberjs/guides/pull/1069

I attempted the rebase and think I messed things up so I just reforked and made a single commit. I'll definitely ask you next time before I try it. :) Thanks!

**Pull request closed without merging**